src/Components/Publishing/Header/FeatureHeader.tsx

```diff
@@ -60,16 +61,24 @@ function renderTextLayoutAsset(url, layout, title, imageChild) {
           <Video src={url} autoPlay controls={false} loop muted playsInline />
         </TextAsset>
       )
-    } else {
+    } else if (layout !== "basic") {
       const alt = url.length ? title : ""
       const src = url.length && resize(url, { width: 1200 })
-      const image = <Image src={src} alt={alt} />
-      return (
-        <TextAsset>
-          {imageChild}
-          {url.length && image}
-        </TextAsset>
-      )
+
+      // When toggling between layout states, guard 'Basic' video headers stored
+      // in temporary `hero_section` auto-save state. TODO: might want to revist.
+      if (!isValidVideoUrl(url)) {
```

Comment:
I think mongo validation should do this already, but we should probably check on the positron side and make sure fields are not being saved if they aren't used.

Reply:
I'm certain that it wasn't guarding against images / other urls, but will follow up in Positron